data/data-pipeline/etl/score/etl_score_geo.py

```diff
@@ -0,0 +1,168 @@
+import pandas as pd
+import geopandas as gpd
+import math
+
+from etl.base import ExtractTransformLoad
+from utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class GeoScoreETL(ExtractTransformLoad):
+    """
+    A class used to generate per state and national GeoJson files with the score baked in
+    """
+
+    def __init__(self):
+        self.SCORE_GEOJSON_PATH = self.DATA_PATH / "score" / "geojson"
+        self.SCORE_LOW_GEOJSON = self.SCORE_GEOJSON_PATH / "usa-low.json"
+        self.SCORE_HIGH_GEOJSON = self.SCORE_GEOJSON_PATH / "usa-high.json"
+
+        self.SCORE_CSV_PATH = self.DATA_PATH / "score" / "csv"
+        self.TILE_SCORE_CSV = self.SCORE_CSV_PATH / "tiles" / "usa.csv"
+
+        self.CENSUS_USA_GEOJSON = (
+            self.DATA_PATH / "census" / "geojson" / "us.json"
+        )
+
+        self.TARGET_SCORE_NAME = "Score E (percentile)"
+        self.TARGET_SCORE_RENAME_TO = "E_SCORE"
+
+        self.NUMBER_OF_BUCKETS = 10
+
+        self.geojson_usa_df: gpd.GeoDataFrame
+        self.score_usa_df: pd.DataFrame
+        self.geojson_score_usa_high: gpd.GeoDataFrame
+        self.geojson_score_usa_low: gpd.GeoDataFrame
+
+    def extract(self) -> None:
+        logger.info(f"Reading US GeoJSON (~6 minutes)")
+        self.geojson_usa_df = gpd.read_file(
+            self.CENSUS_USA_GEOJSON,
+            dtype={"GEOID10": "string"},
+            usecols=["GEOID10", "geometry"],
+            low_memory=False,
```

Comment:
what happens if you pass true 👀

Reply:
It will load in batches, thus much more slowly.